code/game/machinery/alarm.dm

```diff
@@ -307,6 +307,10 @@
 		mode = AALARM_MODE_SCRUBBING
 		apply_mode()
 
+	if(mode == AALARM_MODE_REFILL && environment_pressure > ONE_ATMOSPHERE - 10)

```

Comment:
If you're going to do this (which I don't personally think is needed since someone using Refill is likely monitoring the area anyways, plus the effectiveness of this is highly dependent on the positioning of vent(s) and the air alarm), I'd make this instead run off the warning pressure setting of the air alarm and not standard atmosphere pressure.

Let's say for example you decided you want to keep a room at a pressure of 175kPa for some reason (not sure WHY you'd want that, but we'll assume its probably some devious purpose to keep sec from barging in the front door) and set the warning pressure to anything between 150-173kPa and danger as being anything below 150kPa. If I set the room up to be that way and then try to use refill to reach that intended pressure, it would shut itself off automatically whenever the pressure is above 91.3kPa... well below my target pressure and will still report danger alarms, potentially leading to the mistaken belief that the refill mode doesn't work since it shuts itself off immediately (or on the next process cycle, so within roughly 2 seconds or so). Yes, if someone is going to the trouble of messing with the alarm settings to the point of touching warning and danger levels, they probably know how to change the output pressure of a vent, but the point is that this leads to inconsistent behavior that would pigeonhole this mode into relative uselessness.

Honestly, this feels unnecessary to me, since using Refill usually involves someone monitoring the situation, plus the reliability of the pressure reported by the air alarm can vary wildly between areas based on positioning of vents and the air alarm (air alarms only check the atmos on their tile, so being too close to a vent can lead to major bouncing between the high and low pressure, and being too far away can mean it won't switch off until the room is already going to end up over pressurized)

Just my thoughts on the matter.

Reply:
I'll fix it. For the most part of what you said, I disagree. There has been multiple occasion of atmospherics leaving refill on thinking that it would solve the problem. There is no guide, no warning, or even a hint about what Refill actually does unless you have extensive experience with Air Alarm. 

This PR will not make Refill more or less useless. Manual override will always be an option, but Refill needs work to give Atmos a break from having to monitor it entirely when they could be doing something else.

Reply:
If your concern is that it doesn't communicate the exact effects of the mode well, why not simply update the description it gets in the UI?

In the ui_data proc, you can change the description to say "Triples vent output to fill rooms quickly." instead of the current "Restores air in the room quickly." (which frankly is already a fair hint that it pushes a lot of air in my opinion)